internal/db/migrations-tern/012_add_blueprints_deleted_flag.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE blueprints ADD deleted boolean NOT NULL DEFAULT FALSE;
```

Comment:
Looks safe, @lzap, WDYT? :)

Reply:
Yes, this is definitely correct. There is no point in creating index on a boolean column, it will almost never be used as table scan is usually faster on this type.